satellite/satellitedb/usagerollups.go

```diff
@@ -177,6 +178,153 @@ func (db *usagerollups) GetBucketUsageRollups(ctx context.Context, projectID uui
 	return bucketUsageRollups, nil
 }
 
+// GetBucketTotals retrieves bucket usage totals for period of time
+func (db *usagerollups) GetBucketTotals(ctx context.Context, projectID uuid.UUID, cursor console.BucketUsageCursor, since, before time.Time) (*console.BucketUsagePage, error) {
+	search := fmt.Sprintf("%s%%", cursor.Search)
+
+	if cursor.Limit > 50 {
+		cursor.Limit = 50
+	}
+	if cursor.Page == 0 {
+		return nil, errs.New("page can not be 0")
```

Comment:
Don't pages usually start at `0`?

Reply:
I was not sure where should I add one on the front-end part, or to make it start from 1 on the back-end. If the second option is more correct I can do it that way

Reply:
I'm unsure. Whatever is the standard in graphql